geoextent/lib/handleGeojson.py

```diff
@@ -0,0 +1,225 @@
+
+import json
+import sys
+import datetime
+import logging
+from django.utils.dateparse import parse_datetime
```

Comment:
I've commented on this before: `django` is a heavy dependency to include for only parsing of date and time.

Stongly suggest to switch to `dateutil`, https://dateutil.readthedocs.io/en/stable/ which does

> Generic parsing of dates in almost any string format;

which is exactly what we need. If `dateutil` can parse it, then we accept it. If it raises an error, then it is not a timestamp.

Reply:
Looking at https://stackabuse.com/converting-strings-to-datetime-in-python/, there are also `maya` and `arrow` - please briefly look around to decide which lib is most suitable for us (i.e. great a parsing, but not to heavy).

See also https://opensource.com/article/18/4/python-datetime-libraries

Performance is not so important, and we also don't need "human readable outputs" that much. "stable" is more important than "new".

Reply:
* Using dateutil insteed of django in geojson handling, 
   - Adding function to check if date is valid.
   - Using dateutil to convert string to date.
   - Removing django and iso8601 from requirements.txt